torchgeo/models/swin.py

```diff
@@ -8,19 +8,35 @@
 import kornia.augmentation as K
 import torch
 import torchvision
+from kornia.contrib import Lambda
 from torchvision.models import SwinTransformer
 from torchvision.models._api import Weights, WeightsEnum
 
 from ..transforms import AugmentationSequential
 
 __all__ = ["Swin_V2_B_Weights"]
 
-
 # https://github.com/allenai/satlas/blob/bcaa968da5395f675d067613e02613a344e81415/satlas/cmd/model/train.py#L42 # noqa: E501
-# All Satlas imagery is uint8 and normalized to the range (0, 1) by dividing by 255
+# Satlas Sentinel-1 and RGB Sentinel-2 and NAIP imagery is uint8 and is normalized to (0, 1) by dividing by 255. # noqa: E501
 _satlas_transforms = AugmentationSequential(
-    K.CenterCrop(256),
-    K.Normalize(mean=torch.tensor(0), std=torch.tensor(255)),
+    K.Normalize(mean=torch.tensor(0), std=torch.tensor(255)), data_keys=["image"]
+)
+
+# Satlas multispectral Sentinel-2 imagery divides first 3 bands by 255 and the following 6 bands by 8160, both clipped to (0, 1). # noqa: E501
```

Comment:
Why do we divide B02, B03 and B04 by 255 here and the rest by 8160? I think RGB S2 values are much larger than 255.

Reply:
Ahh I found the details here https://github.com/allenai/satlas/blob/main/Normalization.md#sentinel-2-images. Satlas uses the TCI product for RGB which is already [0,255]. We should definitely link to this / explain it as this will be a bottleneck for getting useful embeddings out of the model!

Reply:
Nice, thanks!